frontend/src/components/molecules/SocialList/SocialList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Social from './Social';
+import { StyledImage } from './Social/Social.styled';
+import { ISocialList } from './SocialList.model';
+import { StyledAllSocialList, StyledName, StyledUserSocialList } from './SocialList.styled';
+
+const SocialList = ({ userSocials, allSocials }: ISocialList): JSX.Element => {
+  return (
```

Comment:
We are using allSocials only in this component. Please retrieve them inside this component instead of retrieving them on the parent side and just drill to child